netbox/circuits/api/serializers.py

```diff
@@ -17,12 +17,17 @@
 class ProviderSerializer(PrimaryModelSerializer):
     url = serializers.HyperlinkedIdentityField(view_name='circuits-api:provider-detail')
     circuit_count = serializers.IntegerField(read_only=True)
+    contacts = NestedContactAssignmentSerializer(
```

Comment:
I'm afraid this approach of following the reverse relationship is not tenable, as it can result in a substantial performance hit. Notice that as a rule we don't follow reverse relations for any model in the REST API (though there may be a handful of exceptions).

REST API consumers should instead query the contact assignments endpoint to retrieve these objects.

Reply:
I have removed the API changes from the PR.